src/Form.tsx

```diff
@@ -386,6 +386,7 @@ const _Form: Form = React.forwardRef(
     };
 
     const handleSubmitSuccess = (validatedValue: InferType<T>) => {
+      notify(onError, []);
```

Comment:
maybe only if there are errors? or maybe it should be more finely grained and only clear the errors that were queued when submit started? IDK, clearing errors on a submit seems like reasonable behavior so probably over thinking it

Reply:
i'm paralleling the logic from https://github.com/taion/react-formal/blob/a68c2b13c77dbdac2ba2e0daeb691291f0d63fb6/src/Form.tsx#L420 , which overwrites all the errors with the validation errors from https://github.com/taion/react-formal/blob/a68c2b13c77dbdac2ba2e0daeb691291f0d63fb6/src/Form.tsx#L456-L460